content/plugins/scm-scw-plugin/plugin.yml

```
@@ -0,0 +1,6 @@
+name: scm-scw-plugin
+displayName: Secure Code Warrior
+description: Secure Code Warrior support
+category: workflow
```

Should be "Workflow" after the latest changes.
```suggestion
category: Workflow
```